pypff-stubs/pypff.pyi

```diff
@@ -0,0 +1,193 @@
+import datetime
+from typing import Any, Iterator, Iterable, Optional
+"""
+The stubs file for pypff, a python wrapper for libpff
```

Comment:
It might be nice to divide this file into sections based on where these definitions came from - that would make it much easier to update the stubs whenever we need to sync our fork to upstream.

This could be as simple as organizing this file like so:
```python
# pypff_file

class file:
    ...
    def signal_abort(self) -> None:
        """Signals the file to abort the current activity."""
    ...

...

# pypff_folder
...
```

Reply:
[41ba7dd](https://github.com/Everlaw/libpff/pull/5/commits/41ba7dd80446ea961ecd2e7aa1448d1428426f13)